domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ioc/InitiatePaymentRequest.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.cmc.domain.models.ioc;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.cmc.domain.models.Interest;
+import uk.gov.hmcts.cmc.domain.models.amount.Amount;
+
+import java.util.UUID;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class InitiatePaymentRequest {
```

Comment:
Did we get answer on whether we can use the full claim data instead this partial?

Reply:
Not yet :(